tools/assets/build.sh

```diff
@@ -30,6 +30,7 @@ function build {
   fi
 
   touch buildLock
+  chmod 664 buildLock
```

Comment:
Ah, why did you need this? A problem of permission depending on which user you use to lunch the asset building maybe?